x-pack/test/security_solution_cypress/cypress/screens/breadcrumbs.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const RULES_TABLE_BREADCRUMB =
+  '[data-test-subj~="breadcrumb"][title="Detection rules (SIEM)"]';
+
+export const LAST_BREADCRUMB = `[data-test-subj~="last"]`;
```

Comment:
Can we make this selector more specific?

Reply:
Sure